### ISSUE TITLE
Create ODBC example

### ISSUE THREAD

**Issue by snowch** [snowch/biginsight-examples#45]
We could possible automate creating the odbc.ini file:

```
[BIGSQL]
Security=ssl
DATABASE=BIGSQL
HOSTNAME=ehaasp-XXX-master-1.bi.services.bluemix.net
PORT=51000
SSLServerCertificate=C:\cert\server\mydbserver.arm
UID=dsbsdes
PWD=XXXXX
```

Some tips from a previous installation:

> BigSQL in v4.1 has been installed with SSL. In order to connect to it, you need the arm file used in the database (I requested via ticket) . It's on /home/bigsql/sqllib/security/keystore/ but biadmin has not rights to read it.  

 **Question** - is this certificate different from the https certificate that we use to connect to BigSQL with Jdbc?

Note that the odbc drivers need to be manually downloaded:
- https://www.ibm.com/support/knowledgecenter/SSPT3X_4.1.0/com.ibm.swg.im.infosphere.biginsights.analyze.doc/doc/bsql_odbc_windows.html?lang=en
- https://www.ibm.com/support/knowledgecenter/SSPT3X_4.1.0/com.ibm.swg.im.infosphere.biginsights.analyze.doc/doc/bsql_odbc_linux.html?lang=en

I don't think this will make a good example because there is very little that can be automated.  If the linux odbc drivers could be made available publicly, we could make an automated example using the Vagrant environment.

**Comment by snowch** [comment]
@pregazzoni - any thoughts on this?

**Comment by pregazzoni** [comment]
Agree, might be too complex to fully automate. Could be done with some pre-req eg.:
1. get odbc driver and provide local path
2. get arm file and provide local path

**Comment by pregazzoni** [comment]
and seems like permission of .arm file might be an issue:
`-r-------- 1 bigsql hadoop 1911 Apr 19 10:35 mydbserver.arm`
